Monika-After-Story/MonikaModDev
# Update thumbnails

Updates all our hair and outfit thumbnails to use the new desk and uniform

Also fixes the front hair for the low bun.

# Testing
* Verify the thumbs looks good
* Verify all the outfit thumbs use the current desk
* Verify all the hair thumbs, except the down tied strand, use the current uniform
* Verify that the thumbs for the outfits that have an outfit mode are complete (aka no missing acs, correct ribbon etc)
